static/js/redux/ui/side_scroller.jsx

```diff
@@ -24,11 +24,6 @@ class SideScroller extends React.Component {
   }
 
   render() {
-    if (this.props.content.length <= 2) {
-      return (
-        <div style={{ marginBottom: "-30px !important" }}>{this.props.content}</div>
-      );
-    }
```

Comment:
this block doesn't seem to be useful at all, does not render the content correctly and shows empty